src/components/styles/InputField.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
```

Comment:
this whole file makes no sense to me. You name it InpuField.js but there is no StyledInput, but a label and textarea...

Reply:
I do not see, that this comment is resolved, as the file has not been changed. I agree that you could improve your naming for better readability for externals - but up to you